src/nunit.gui.core/Model/Settings/GuiSettings.cs

```diff
@@ -59,18 +59,11 @@ public class MainFormSettings : SettingsWrapper
     {
         public MainFormSettings(ISettings settingsService) : base(settingsService, "Gui.MainForm") { }
 
-        private const string locationKey = "Location";
-        public Point Location
+        private const string windowShapeKey = "LocationAndSize";
+        public WindowShape LocationAndSize
```

Comment:
no, i just figured it was worth combining. I forget what it it is off the top of my head in WPF, but I think the concept is the same as SWF